tests/integration/targets/eks_nodegroup/tasks/dependecies.yml

```diff
@@ -106,3 +106,20 @@
     default_version: 1
     instance_type: t3.micro
   register: lt
+
+- name: Create securitygroup for node access
+  amazon.aws.ec2_security_group:
+    name: 'ansible-test-eks_nodegroup'
+    description: "SSH access"
+    vpc_id: '{{ setup_vpc.vpc.id }}'
+    rules:
+      - proto: tcp
+        ports:
+          - 22
+        cidr_ip: 0.0.0.0/0
+  register: securitygroup_eks_nodegroup
+
+- name: Create Keypair for Access to Nodegroup nodes
+  amazon.aws.ec2_key:
+    name: "ansible-test-eks_nodegroup"
+  register: ec2_key_eks_nodegroup
```

Comment:
@romulus-ai can you also add both ressources to the `cleanup.yml` to take care, they are removed after testing is finished?  
otherwise looks good to me!

Reply:
Hey, yes sure, forgot that, sry.

Reply:
@markuman its already in cleanup.yml: [tests/integration/targets/eks_nodegroup/tasks/cleanup.yml](https://github.com/ansible-collections/community.aws/pull/1773/files#diff-6f07b6f67c94c2f9da418468d2f2cca270b4538442e38f4d148021f820bbf09cR40)